plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchange.java

```diff
@@ -67,7 +67,7 @@
         implements Exchange
 {
     private static final Pattern PARTITION_FILE_NAME_PATTERN = Pattern.compile("(\\d+)_(\\d+)\\.data");
-    private static final char[] RANDOMIZED_PREFIX_ALPHABET = "abcdefghijklmnopqrstuvwzyz0123456789".toCharArray();
+    private static final char[] RANDOMIZED_PREFIX_ALPHABET = "abcdef0123456789".toCharArray();
```

Comment:
Please add a comment explaining why this exact alphabet is important.

Reply:
Discussed offline